DEVELOPMENT.md

```diff
@@ -40,6 +40,11 @@ psql -h localhost -U postgres -p 5432 postgres < sippy-backup-2022-05-02.sql
 
 or with a command like this if you are using the custom format file:
 
+Optional - If it's a new db then initialize roles (or just ignore the errors):
+```bash
+echo "CREATE USER sippyro;" | psql postgresql://postgres:password@localhost:5432/postgres
+echo "CREATE USER rdsadmin;" | psql postgresql://postgres:password@localhost:5432/postgres
+```
```

Comment:
The accounts we use probably should not be appearing in public docs esp if the errors are harmless.